addons/website/static/src/snippets/s_accordion/000.scss

```diff
@@ -0,0 +1,69 @@
+.s_accordion {
+
+    // Replace the default chevron with our Odoo UI icon to
+    // have more control on its color depending on the
+    // preset applied
+    .accordion-button:after {
+        margin: 0;
+        background: none;
+        font-family: 'odoo_ui_icons';
+        text-align: center;
+        content: '\e839';
+    }
+
+    .s_accordion_highlight {
+        .accordion-item {
+            border: 0;
+            border-radius: var(--#{$prefix}accordion-border-radius);
+            overflow: hidden;
```

Comment:
The `overflow: hidden` is necessary to clip the entire item when you set a custom background color. Otherwise there would be a rounded corner issue:

| Without overflow | With overflo |
|--------|--------|
| <img width="805" alt="Capture d’écran 2024-08-08 à 14 12 27" src="https://github.com/user-attachments/assets/ad678d7e-6a93-4c3e-a81c-5ac73406f703"> | <img width="820" alt="Capture d’écran 2024-08-08 à 14 13 16" src="https://github.com/user-attachments/assets/612c90c1-bbc0-4729-908e-3f9f3e07eb81"> |

Reply:
I've updated the branch to improve the focus state ;)